### ISSUE TITLE
The BMF calendar uses a non-existent timezone.

### ISSUE THREAD

**Issue by ssanderson** [quantopian/zipline#1393]
```
In [2]: get_calendar('BMF')
---------------------------------------------------------------------------
UnknownTimeZoneError                      Traceback (most recent call last)
<ipython-input-2-83acbbe233c7> in <module>()
----> 1 get_calendar('BMF')

/home/ssanderson/quantopian/qexec/zipline_repo/zipline/utils/calendars/calendar_utils.pyc in get_calendar(self, name)
     77                 # Use the same calendar for all exchanges that share the same
     78                 # factory.
---> 79                 calendar = factory()
     80                 self._calendars.update({n: calendar for n in names})
     81                 return calendar

/home/ssanderson/quantopian/qexec/zipline_repo/zipline/utils/calendars/trading_calendar.pyc in __init__(self, start, end)
     63
     64         # `DatetimeIndex`s of standard opens/closes for each day.
---> 65         self._opens = days_at_time(_all_days, self.open_time, self.tz,
     66                                    self.open_offset)
     67         self._closes = days_at_time(

/home/ssanderson/quantopian/qexec/zipline_repo/zipline/utils/calendars/exchange_calendar_bmf.pyc in tz(self)
    203     @property
    204     def tz(self):
--> 205         return timezone("America/Sao_Paolo")
    206
    207     @property

/home/ssanderson/.virtualenvs/qexec2/local/lib/python2.7/site-packages/pytz/__init__.pyc in timezone(zone)
    178                 fp.close()
    179         else:
--> 180             raise UnknownTimeZoneError(zone)
    181
    182     return _tzinfo_cache[zone]

UnknownTimeZoneError: 'America/Sao_Paolo'
```

I assume this means it's untested.

**Comment by richafrank** [comment]
Unrelated to the issue, but we're defining the list of holidays twice in that module!

**Comment by ssanderson** [comment]
Looks like this should be `Sao_Paulo`:

```
In [12]: pd.Timestamp('2014-01-01', tz='America/Sao_Paulo')
Out[12]: Timestamp('2014-01-01 00:00:00-0200', tz='America/Sao_Paulo')
```

**Comment by richafrank** [comment]
Ah nice - that matches the value from before the most recent changes.

**Comment by richafrank** [comment]
Should be fixed by #1418.
